account_product_move/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+To use this module, you need to:
+
+#. Go to *Accounting > Accounting > Journals > Journal Templates*.
+#. Create a new Account Product Move record and give it a name.
+#. Create product template lines.
+#. Create Product Move Lines for these products.
+#. Create a new out invoice with product lines.
+#. Post the out invoice.
+#. If any of the products in invoice lines has an Account Product Move record set-up, exist a new journal entry will be added per line in Account Product Move Lines of Account Product Move, holding these Account Product Move Lines.
```

Comment:
Maybe the last three steps more clear as:

#. Create a new outgoing invoice with invoice lines mentioning some of these products.
#. Confirm the invoice.
#. A new journal entry will be added per line in Account Product Move Lines of Account Product Move, holding these Account Product Move Lines.